GeeksCoreLibrary/Modules/Databases/Services/MySqlDocumentStorageService.cs

```diff
@@ -0,0 +1,113 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using GeeksCoreLibrary.Core.DependencyInjection.Interfaces;
+using GeeksCoreLibrary.Core.Models;
+using GeeksCoreLibrary.Modules.Databases.Interfaces;
+using GeeksCoreLibrary.Modules.Databases.Models;
+
+namespace GeeksCoreLibrary.Modules.Databases.Services;
+
+/// <inheritdoc />
+public class MySqlDocumentStorageService : IDocumentStorageService, IScopedService
+{
+    private readonly IDocumentStoreConnection documentStorageConnection;
+
+    public MySqlDocumentStorageService(IDocumentStoreConnection documentStorageConnection)
+    {
+        this.documentStorageConnection = documentStorageConnection;
+    }
+
+    /// <inheritdoc />
+    public async Task CreateCollection(string prefix = "")
```

Comment:
Please update the name to `CreateCollectionAsync` (although this will probably happend automatically if you refactor the name in the interface).